### PR TITLE
Added CmsBlockGroupTestCase

### PR DESCRIPTION
**Pull request by JohnnyMcWeed**
PR for issue: https://github.com/luyadev/luya-testsuite/issues/8

### CODE REVIEW

File: src/cases/CmsBlockGroupTestCase.php [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        }
+
+        $class = $this->blockGroupClass;
+        $this->blockGroup = new $class();
+    }
+
+    /**
+     * Tests if the identifier is correct
+     */
+    public function testIdentifier()
```
Comment (nadar) [inline src/cases/CmsBlockGroupTestCase.php:91]: If someone wants to check if those test are working, and not having a risky test you could just return true.

So you could do $this->assertTrue($this->testIdentifier()); to make sure tests are not risky.

same for testLabel and testGetPosition

**Review by nadar — CHANGES_REQUESTED** [review]
tester shold return true so people could implement this in order to mark test not as risky.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file. This project make usage of the [Yii Versioning Strategy](https://github.com/yiisoft/yii2/blob/master/docs/internals/versions.md).
 
+## 1.0.11 (in progress)
++ [#8](https://github.com/luyadev/luya-testsuite/issues/8) Added class/new feature: CmsBlockGroupTestCase
```
Comment (nadar) [inline CHANGELOG.md]: Tiny adjustment: Add newline after title and mabye change message to: Added new CmsBLockGroupTestCase in order to test block groups. 

or sth. like this

File: tests/data/TestGroup.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace luya\testsuite\tests\data;
+
+final class TestGroup
```
Comment (nadar) [inline tests/data/TestGroup.php]: The test group should extend from https://luya.io/api/luya-cms-base-BlockGroup

File: src/cases/CmsBlockGroupTestCase.php [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+            throw new InvalidConfigException("The 'blockGroupClass' property can not be empty.");
+        }
+        if (!$this->blockGroupIdentifier) {
+            throw new InvalidConfigException("The 'blockGroupIdentifier' property can not be empty.");
+        }
+
+        $class = $this->blockGroupClass;
+        $this->blockGroup = new $class();
+    }
+
```
Comment (nadar) [inline src/cases/CmsBlockGroupTestCase.php:78]: Another test should be to check whether the object is instance of https://luya.io/api/luya-cms-base-BlockGroup

**Review by nadar — CHANGES_REQUESTED** [review]

File: tests/data/TestGroup.php (hunk 3 above)
Comment (JohnnyMcWeed) [reply tests/data/TestGroup.php]: I completely agree with you. This should be done in others, too. See: 
- https://github.com/luyadev/luya-testsuite/blob/master/tests/data/TestBlock.php
- https://github.com/luyadev/luya-testsuite/blob/master/tests/cases/CmsBlockTestCaseTest.php

I can add it if you want..

File: src/cases/CmsBlockGroupTestCase.php (hunk 4 above)
Comment (JohnnyMcWeed) [reply src/cases/CmsBlockGroupTestCase.php:78]: I completely agree with you. This should be done in others, too. See: 
- https://github.com/luyadev/luya-testsuite/blob/master/tests/data/TestBlock.php
- https://github.com/luyadev/luya-testsuite/blob/master/src/cases/CmsBlockTestCase.php

I can add it if you want..

Comment (nadar) [reply src/cases/CmsBlockGroupTestCase.php:78]: We could, but maybe in another issue/PR.

**Pull request merged**